NuGet.Config

```diff
@@ -4,5 +4,11 @@
     <add key="enabled" value="True" />
     <add key="automatic" value="True" />
   </packageRestore>
-  <!-- Note: Do not specify any NuGet feeds in this file, everything is available on the fallback NuGet.org. -->
+  <packageSources>
+    <clear />
+    <!-- Warning: Never add more NuGet feeds, to be compliant this repo must only rely on this single ADO feed. -->
+    <add key="react-native" value="https://pkgs.dev.azure.com/ms/react-native/_packaging/react-native-public/nuget/v3/index.json" />
+    <!-- Remove (comment out) the nuget.org feed once our ADO feed is configured correctly. See https://github.com/microsoft/react-native-windows/issues/11557 -->
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
Just FYI: Nuget fires requests to all feeds at the same time and first to return will be chosen. If you want ot populate our public feed you'll need ot make sure you have no nuget cache and remove this line...

Reply:
Yeah, the goal is once we're ready to turn on upstream for `react-native-public` then I'll remove the nuget.org line and going forward we'll be compliant.